core/src/main/java/nl/nn/adapterframework/util/XmlUtils.java

```diff
@@ -961,21 +938,28 @@ public static Source stringToSource(String xmlString) throws DomBuilderException
 	public static Source stringToSourceForSingleUse(String xmlString) throws SAXException {
 		return stringToSourceForSingleUse(xmlString, isNamespaceAwareByDefault());
 	}
-	
+
 	public static Source stringToSourceForSingleUse(String xmlString, boolean namespaceAware) throws SAXException {
-		StringReader reader = new StringReader(xmlString);
-		InputSource is = new InputSource(reader);
-		return inputSourceToSAXSource(is, namespaceAware, null);
-	}	
-	
+		if (namespaceAware) {
+			StringReader reader = new StringReader(xmlString);
+			InputSource is = new InputSource(reader);
+			return inputSourceToSAXSource(is, namespaceAware, null);
```

Comment:
Waarom? Wat is het punt van een InputSource als je niets streaming doet, dan kan het net zo goed altijd een domsource zijn?

Reply:
XSLT 1 is wel streaming